src/PKSim.Core/Services/HIDiseaseStateImplementation.cs

```diff
@@ -41,8 +41,8 @@ public static class ChildPughScore
       private static readonly HIFactors _portalFlowScalingFactor = createFactors(0.4, 0.36, 0.04);
       private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.3, 2.3, 3.4);
       private static readonly HIFactors _hepaticVolumeScalingFactor = createFactors(0.69, 0.55, 0.28);
-      private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.65);
-      private static readonly HIFactors _otherOrgansFlowScalingFactor = createFactors(1.31, 1.84, 2.27);
+      private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.48);
+      private static readonly HIFactors _cardiacIndexDiseaseFactor = createFactors(1.11, 1.27, 1.36);
```

Comment:
did not check the number changes - assume they are correct :)

Reply:
should `_cardiacIndexDiseaseFactor ` be named `_cardiacIndexScalingFactor` to be consistent with all other factors?

Reply:
I just sticked with the implementation nomenclatur but yeah probably makes sense.